content/actions/guides/building-and-testing-powershell.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Building and testing PowerShell
+intro: You can create a continuous integration (CI) workflow to build and test your PowerShell project.
+versions:
```

Comment:
This is something that was also missed in the Python guide, but we're updating that in another PR. So I wouldn't expect you to have included it here. 

It just displays this banner, which you'll see in the other actions pages:
<img width="672" alt="Screen Shot 2020-10-09 at 4 11 41 PM" src="https://user-images.githubusercontent.com/9831992/95638638-2cc68f00-0a4a-11eb-8075-af93d2d4bdf4.png">




```suggestion
product: '{% data reusables.gated-features.actions %}'
versions:
```